src/components/global/Header/Header.module.scss

```diff
@@ -73,7 +73,8 @@
     align-items: center;
     justify-content: center;
     height: 7.3rem;
-    background-color: var(--black-01);
+    background-color: var(--white);
+    border-bottom: 1px var(--gray-01) solid;
```

Comment:
A small nitpick, isn't it `1px solid var(--gray-01);`? Looks like you flipped the color and the line type (solid). Does it still work like this?

Reply:
NVM, I just checked on the deployment link, looks like it does